library/core/src/slice/mod.rs

```diff
@@ -2585,7 +2585,8 @@ impl<T> [T] {
     /// buf.fill(1);
     /// assert_eq!(buf, vec![1; 10]);
     /// ```
-    #[unstable(feature = "slice_fill", issue = "70758")]
+    #[doc(alias = "memset")]
+    #[stable(feature = "slice_fill", since = "1.51.0")]
```

Comment:
This should be able to make it for 1.50. (1.50 will move to beta December 31st.)

```suggestion
    #[stable(feature = "slice_fill", since = "1.50.0")]
```

Reply:
Oh my bad, thank you! -- updated!